docs/src/main/paradox/scala/cluster-http-management.md

```diff
@@ -0,0 +1,202 @@
+<a id="http-cluster-management"></a>
+# Cluster Http Management
+
+Akka Cluster HTTP Management is module that allows you interaction with an `akka-cluster` through an HTTP interface.
+This module exposes different operations to manage nodes in a cluster. 
+
+The operations exposed are comparable to the Command Line Management tool or the JMX interface `akka-cluster` provides.
+
+## Preparing your project for Cluster HTTP Management
+
+The Akka Cluster HTTP Management is a separate jar file. Make sure that you have the following dependency in your project::
+
+  "com.typesafe.akka" %% "akka-cluster-http-management" % "@version@" @crossString@
```

Comment:
I think it should be:

    "com.typesafe.akka" %% "akka-cluster-http-management" % "$version$"